columnflow/columnar_util.py

```diff
@@ -531,6 +531,18 @@ def apply(
         return res
 
 
+class ColumnCollection(enum.Flag):
+    """
+    Enumeration containing flags that describe arbitrary collections of columns.
+    """
+
+    ALL_FROM_CALIBRATOR = enum.auto()
+    ALL_FROM_CALIBRATORS = enum.auto()
+    ALL_FROM_SELECTOR = enum.auto()
+    ALL_FROM_PRODUCER = enum.auto()
+    ALL_FROM_PRODUCERS = enum.auto()
+
```

Comment:
you could also add one `ALL_FROM_ML_EVALUATION` ColumnCollection

Reply:
Good point. Added :+1: